src/Akeneo/Connectivity/Connection/front/src/connect/components/ConnectedApps/ConnectedAppsContainer.tsx

```diff
@@ -37,6 +38,10 @@ export const ConnectedAppsContainer: FC<Props> = ({connectedApps}) => {
     const ref = useRef(null);
     const scrollContainer = findScrollParent(ref.current);
     const displayScrollButton = useDisplayScrollTopButton(ref);
+
+    const connectedTestApps = connectedApps.filter((connectedApp: ConnectedApp) => connectedApp.test);
+    connectedApps = connectedApps.filter((connectedApp: ConnectedApp) => !connectedApp.test);
```

Comment:
maybe name this variable differently to be more explicit ? `realConnectedApps` ? or rename the prop : 
```suggestion
    connectedApps = allConnectedApps.filter((connectedApp: ConnectedApp) => !connectedApp.test);
```